src/MFRC522.cpp

```diff
@@ -364,6 +364,38 @@ bool MFRC522::PCD_PerformSelfTest() {
 } // End PCD_PerformSelfTest()
 
 /////////////////////////////////////////////////////////////////////////////////////
+// Power control
+/////////////////////////////////////////////////////////////////////////////////////
+
+//IMPORTANT NOTE!!!!
+//Calling any other function that uses CommandReg will disable soft power down mode !!!
+
+void MFRC522::power_down()//Note : Only soft power down mode is available throught software
```

Comment:
CS (code style): please like the other functions: `void MFRC522::power_down() {`